src/kitFilterHelpers.ts

```diff
@@ -0,0 +1,56 @@
+import { generateHash } from "./utils";
+// TODO: EventType/EventTypeEnum exists in differnet forms between @types/eventmodel/dataplanningnode.  determine the differences and consolidate if possible
+import { EventTypeEnum } from "./types.interfaces";
+import { UserIdentityType } from "./forwarders.interfaces";
+import { IdentityType } from "@mparticle/web-sdk";
+
+
+export default class KitFilterHelpers {
```

Comment:
Nitpick about the name of this class. Is it really a helper and is it limited to only kits? Unless this is the naming convention we're using across all SDKs, maybe we call it `HashUtility`?

Reply:
@einsteinx2 @markvdouw - We should discuss this since it is a cross platform naming topic.

Reply:
Let's go with `KitFilterHelper` singular